discovery-provider/docker-compose.yml

```diff
@@ -3,15 +3,66 @@ version: "3.9"
 services:
   db:
     container_name: postgres
+    image: postgres:${audius_pg_upgrade_to_version:-11.22-bookworm}
+    shm_size: 2g
+    restart: always
+    entrypoint: >
+        /bin/bash -c 
+        "if [ -f /var/lib/postgresql/data/pg_hba.conf ]; then
+          if [[ $$(tail -n 1 /var/lib/postgresql/data/pg_hba.conf) != 'hostnossl    all          all            0.0.0.0/0  trust' ]]; then
+            echo 'hostnossl    all          all            0.0.0.0/0  trust' >> /var/lib/postgresql/data/pg_hba.conf;
```

Comment:
does this allow unrestricted access?

Reply:
it allows unrestricted access, but the user+pw is already postgres:postgres. no one should allow traffic to port 5432. i made this change because postgres 15 changes the default here, which i've seen block unencrypted traffic from Content Nodes both when containerized and hosted